web/src/hotkey.ts

```diff
@@ -952,9 +965,9 @@ export function process_hotkey(e, hotkey) {
     }
 
     if (event_name === "toggle_compose_preview") {
-        const $last_focused_compose_type_input = $(
-            compose_state.get_last_focused_compose_type_input(),
-        );
+        const last_focused_compose_type_input = compose_state.get_last_focused_compose_type_input();
+        assert(last_focused_compose_type_input !== undefined);
```

Comment:
I don't thinks assertion is accurate. What happens if you open the web app, without accessing compose, and just hit this hotkey? I think we instead just want to `return false` if this is undefined.

Reply:
I tried that, and it still had `last_focused_compose_type_input` as the textarea. But I think returning false makes more sense here since it can be undefined.